src/main/scala/dotty/dokka/model/api/api.scala

```diff
@@ -130,7 +130,7 @@ extension[T] (member: Member):
 
   // TODO rename parent and knownChildren
   def allMembers: Seq[Member] = compositeMemberExt.fold(Nil)(_.members)
-  def parents: Seq[LinkToType] = compositeMemberExt.fold(Nil)(_.parents)
+  def parents: Map[LinkToType, Seq[LinkToType]] = compositeMemberExt.fold(Map())(_.parents)
```

Comment:
I do not thing this is a good change since we lost information about linearization of parents what may be important. I think the simplest solution may be construct graph withn `ClasslikeSupport` and store it within extension or use `Seq[(LinkToType, Seq[LinkToType])]` here (or map that preserve ordering). 

If we go for `Seq` approach we should probably create custom case class instead of using `Tuple`

Reply:
Maybe we should use an ordered map? I'm not sure if preserving the linearization order is very important in the graph, as we anyway don't show it correctly (we show a graph after all, not a list); anyway, the option of using an ordered map is there.